#! /bin/sh /usr/share/dpatch/dpatch-run
## 10_fix_bashims.dpatch by Arjan Oosting <arjan@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fix a bashism in the examples of lhs2TeX

@DPATCH@
Index: lhs2tex-1.15/Examples/lhs2TeXpre
===================================================================
--- lhs2tex-1.15.orig/Examples/lhs2TeXpre	2009-12-18 13:21:18.000000000 +0100
+++ lhs2tex-1.15/Examples/lhs2TeXpre	2010-03-25 12:16:23.000000000 +0100
@@ -5,7 +5,7 @@

 LHSHOME=..

-if [ "$1" == "$2" ]; then
+if [ "$1" = "$2" ]; then
   cp "$2" "$3"
 else
   if grep -q "^%include" "$1"; then
Index: lhs2tex-1.15/doc/lhs2TeXpre
===================================================================
--- lhs2tex-1.15.orig/doc/lhs2TeXpre	2009-12-18 13:21:18.000000000 +0100
+++ lhs2tex-1.15/doc/lhs2TeXpre	2010-03-25 12:16:23.000000000 +0100
@@ -5,7 +5,7 @@

 LHSHOME=..

-if [ "$1" == "$2" ]; then
+if [ "$1" = "$2" ]; then
   cp "$2" "$3"
 else
   if grep -q "^%include" "$1"; then
